routing/localchans/manager.go

```diff
@@ -74,6 +74,12 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 		)
 		edge.TimeLockDelta = uint16(newSchema.TimeLockDelta)
 
+		// MaxHTLC is optional in channel policy update requests.
+		if newSchema.MaxHTLC != 0 {
```

Comment:
Shouldn't this optionality logic be applied for all the other fields? Otherwise, we'll modify the behavior for users which are using the existing API as is.

Reply:
I tried to bring this up in this pr's predecessor #3439. Imo we should either make all fields mandatory (my preference, I think this is good enough) or use flags to signal which fields to update. However, it was then decided that we only wanted to special case max htlc if I didn't misunderstood. 

I don't think the optionality logic as it is presented in this pr breaks the existing api, because the max htlc field didn't yet exist on the api.